Sources/StreamChatUI/UIConfig+Fonts.swift

```diff
@@ -8,6 +8,7 @@ public extension _UIConfig {
     struct Font {
         public var caption1 = UIFont.preferredFont(forTextStyle: .caption1)
         public var footnoteBold = UIFont.preferredFont(forTextStyle: .footnote).bold
+        public var footnote = UIFont.preferredFont(forTextStyle: .footnote)
```

Comment:
Could you explain somehow? Some design changes were made?

Reply:
Yes the figma design now uses footnote for the body of the Last message of the channel list :)